### PR TITLE
Update submissions-report-messages-files-to-microsoft.md

### PR DESCRIPTION
**Pull request by madalenamedo**
Minor change because a new tab was added in the Defender Portal page inside Submissions

### CODE REVIEW

File: microsoft-365/security/office-365-security/submissions-report-messages-files-to-microsoft.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Watch this video that shows more information about the unified submissions exper
...
 |[The Microsoft Report Message and Report Phishing add-ins](submissions-outlook-report-messages.md#use-the-report-message-and-report-phishing-add-ins-in-outlook)|User|These free add-ins work in Outlook on all available platforms. For installation instructions, see [Enable the Report Message or the Report Phishing add-ins](submissions-users-report-message-add-in-configure.md).|
-|[The Submissions page in the Microsoft 365 Defender portal](submissions-admin.md)|Admin|Admins use this method to report good (false positives) and bad (false negative) **Email**, **Email attachments**, and **URLs**, including user reported messages, to Microsoft for further analysis. Available in organizations with Exchange Online mailboxes (not available in standalone EOP).|
+|[The Submissions page in the Microsoft 365 Defender portal](submissions-admin.md)|Admin|Admins use this method to report good (false positives) and bad (false negative) **Email**, **Email attachments**, **URLs** and **Files**, including user reported messages, to Microsoft for further analysis. Available in organizations with Exchange Online mailboxes (not available in standalone EOP).|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-report-messages-files-to-microsoft.md]:
```suggestion
|[The Submissions page in the Microsoft 365 Defender portal](submissions-admin.md)|Admin|Admins use this method to report good (false positives) and bad (false negative) **Email**, **Email attachments**, **URLs**, and **Files**, including user reported messages, to Microsoft for further analysis. Available in organizations with Exchange Online mailboxes (not available in standalone EOP).|
```

Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-report-messages-files-to-microsoft.md:45]:
```suggestion
|[The Submissions page in the Microsoft 365 Defender portal](submissions-admin.md)|Admin|Admins use this method to report good (false positives) and bad (false negative) **Email**, **Email attachments**, **URLs**, and **Files**, including user reported messages, to Microsoft for further analysis. Available in organizations with Exchange Online mailboxes (not available in standalone EOP).|
```

Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-report-messages-files-to-microsoft.md:45]:
```suggestion
|[The Submissions page in the Microsoft 365 Defender portal](submissions-admin.md)|Admin|Admins use this method to report good (false positives) and bad (false negative) **Email**, **Email attachments**, **URLs**, and **Files**, including user reported messages, to Microsoft for further analysis. Available in organizations with Exchange Online mailboxes (not available in standalone EOP).|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @madalenamedo, kindly commit the minor suggested change. Many thanks.

**Comment by madalenamedo** [comment]
@JohanFreelancer9 done! Thank you!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @madalenamedo @JohanFreelancer9

**Pull request merged**